index.html

```diff
@@ -41,19 +41,24 @@
       <!-- /Chatra {/literal} -->
       <title>KendallDoesCoding - Portfolio</title>
    </head>
+
+
+   
    <body>
       <!-- Google Tag Manager (noscript) -->
       <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-TR6RS2G"
       height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
       <!-- End Google Tag Manager (noscript) -->
       <section class="hero">
          <nav>
-            <img class="logo" id="logo" src="https://i.ibb.co/zJG8hYw/gsfyqoyk-0.png" alt="gsfyqoyk-0" border="0"></a>
+            <img class="logo" id="logo" src="/portfolio/images/newlogo.png" alt="gsfyqoyk-0" border="0"></a>

```

Comment:
```suggestion
            <img class="logo" id="logo" src="/images/newlogo.png" alt="KendallDoesCoding-logo" border="0"></a>

```

i think this should fix the issue?

Reply:
Let me again give it a shot...

Reply:
@KendallDoesCoding  yes that will work,

Reply:
Please commit suggestion.